core/trino-main/src/main/java/io/trino/execution/scheduler/TaskExecutionClass.java

```diff
@@ -21,21 +21,30 @@ public enum TaskExecutionClass
     // Tasks from stages with some upstream stages still running.
     // To be scheduled only if no STANDARD tasks can fit.
     // Picked to kill if worker runs out of memory to prevent deadlock.
-    SPECULATIVE;
+    SPECULATIVE,
```

Comment:
this line isn't added in `Introduce EAGER_SPECULATIVE task execution class`, but appears as new in that commit when i run git blame